fe/fe-core/src/main/java/org/apache/doris/tablefunction/S3TableValuedFunction.java

```diff
@@ -113,6 +109,26 @@ public S3TableValuedFunction(Map<String, String> properties) throws AnalysisExce
         }
     }
 
+    private String constructEndpoint(Map<String, String> properties, S3URI s3uri) throws AnalysisException {
```

Comment:
Do we have to check ak sk for all kinds of providers?

Reply:
No need to do this.